chirp/wxui/main.py

```diff
@@ -942,7 +943,7 @@ def make_menubar(self):
         menu_bar.Append(edit_menu, wx.GetStockLabel(wx.ID_EDIT))
         menu_bar.Append(view_menu, '&' + _('View'))
         menu_bar.Append(radio_menu, '&' + _('Radio'))
-        menu_bar.Append(help_menu, wx.GetStockLabel(wx.ID_HELP))
+        menu_bar.Append(help_menu, _('Help'))
```

Comment:
Hmm, we're really not supposed to need to do this for these special items, as the `GetStockLabel` helper accounts for platform differences, like on macos it's common for the "find" issue to be "Find..." but not on others. I think this means it's not translated for your platform, but should be. Which platform are you on? Any chance the formal builds work properly and running from a dev environment does not?

Reply:
Yes, that is strange. I found this on formal build at two Win10 machines :
![image](https://github.com/kk7ds/chirp/assets/2874737/faf122c8-72db-48b9-8226-3b25f92e0bb0)

I'm not sure if this is PL related or Windows related, but definitely something is not working with that configuration. What is interesting rest of the labels (Copy & Paste etc.) were properly pulled via GetStockLabel. 

This is how it looks for me: 
![image](https://github.com/kk7ds/chirp/assets/2874737/164a5981-f662-43be-a029-91ba079bd76d)

Reply:
Hmm, yeah, weird. On Linux (where I can easily flip the locale for one program) I get Help translated by stock item, but not Find, when using the `es` locale. On windows, `es` has both Find and Help translated. I can replicate your experience on windows with `pl`. On macos and `pl`, Help is translated but not find. Ugh!

Okay, I guess this is the best plan then. I'll rebase this and fix the conflicts.

Reply:
Perfect, thanks! 👍🏻